### ISSUE TITLE
Animal Crossing: New Leaf MSBT files blank

### ISSUE THREAD

**Issue by Arutemu64** [IcySon55/Kuriimu#400]
Issue: Kuriimu doesn't show text entries in AC:NL MSBT files.
Console: Nintendo 3DS
Format extension(s): MSBT (exported from UMSBT)
Type: text
Game Name: Animal Crossing
More details:
Back in August, Kuriimu was working with ACNL MSBT files just fine. Now it displays nothing in most MSBT files.

Sample files (if possible):
https://drive.google.com/drive/folders/17cpcM08jdcabK5AxP4StTVKuYj4nkLQF?usp=sharing
These files are dumped directly from my Game Cart
SYS_2D_UI.umsbt - no text entries in every MSBT file exported
Another file has some entries, but I'm not sure they are full.

**Comment by Slattz** [comment]
Is there any updates on this issue? I am also experiencing it, some files work and some don't.

I've noticed that it's only files without a `LBL1` Section that don't work. For example, the file posted above hasn't that section and doesn't work, but this file [here](http://www.mediafire.com/file/zbal2ctsxe90zi0/00000000.msbt) has that section and works fine in Kuriimu.
It also isn't a 'one off', it's consistent that only the files that have the `LBL1` section will work. Hopefully this info will help fix the problem.

**Comment by IcySon55** [comment]
That is actually very useful. The only time I've across files without LBL1 were files from Mario Kart DS. I guess I'll have to figure this mess out then.

**Comment by IcySon55** [comment]
Nope. I've had high priority items forever that are time critical and fixing plugin issues has not been on my plate for quite some time.

**Comment by Slattz** [comment]
Ah ok, that's no problem. I may take a look at it myself.
Just wondering, I assume the plugin just follows a strict structure for msbt, and any file not following that struct is considered invalid?

**Comment by IcySon55** [comment]
It's more like, the code the loads the text entries into the list fails to load them when LBL1 is null because you need to add special handling for when LBL1 is missing. Like naming the entries sequentially for starters. Changes to how the list is created are probably required to make both LBL1 and non-LBL1 support work.

**Comment by Slattz** [comment]
Can confirm this bug has been fixed in commit c1911abc9b2e085ac1120dcc0e01b1c988778638 by @Cuyler36 and this issue can now be closed 🙂
